src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int figureCount = 4;
+    public static final int limit = 100;
+    public static final int defoultRadius = 10;
+    public static final String defoultColor = "white";
+
+    public int getRandomInt() {
+        return new Random().nextInt(limit);
+    }
+
+    public double getRandomDouble() {
+        return new Random().nextDouble();
+    }
+
+    public Figure getDefoultFigure() {
+        return new Circle(defoultColor,defoultRadius);
+    }
+
+    public Figure getRandomFigure() {
+        ColorSuppier colorSuppier = new ColorSuppier();
+        int figureIndex = new Random().nextInt(figureCount);
+        switch (figureIndex) {
+            case 0:
+                return new Circle(colorSuppier.getRandomColor(),getRandomInt());
```

Comment:
```suggestion
                return getRandomCircle();
```
let's create separate private methods, to move creation there
this would make code easier to read